PROCESSORS.md

```diff
@@ -37,6 +37,7 @@
 - [ListSFTP](#listsftp)
 - [ListenHTTP](#listenhttp)
 - [ListenSyslog](#listensyslog)
+- [ListAzureDataLakeStorage](#listazuredatalakestorage)
```

Comment:
nitpick: not in alphabetical order

Reply:
Updated in ef0172d119c472a1c91a10aecbe531a112ade2bf